CONTRIBUTIONS.adoc

```diff
@@ -0,0 +1,269 @@
+= Contribution guidelines
+
+These guidelines instruct how to submit issues and contribute code or
+documentation to the
+https://github.com/robotframework/RIDE[RIDE]. There are also many other projects in the larger
+http://robotframework.org[Robot Framework ecosystem] that you can
+contribute to. If you notice a library or tool missing, there is hardly
+any better way to contribute than creating your own project. Other great
+ways to contribute include answering questions and participating
+discussion on
+https://groups.google.com/forum/#!forum/robotframework-users[robotframework-users]
```

Comment:
Add the preferred Forum (like in README: https://forum.robotframework.org/c/tools/ride/21)